ProjectSetup/Contracts/V1/Requests/CreatePostRequest.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ProjectSetup.Contracts.V1.Requests
+{
+	public class CreatePostRequest
+	{
+		public Guid Id { get; set; }
```

Comment:
Hide this field from front end. Don't allow user to see the ID during category creation.

Reply:
I removed it, thanks